README.md

```diff
@@ -18,7 +18,7 @@ Currently the operator installs the following products:
 - Integreatly solution explorer
 
 ## Prerequisites
-- [operator-sdk](https://github.com/operator-framework/operator-sdk) version v0.15.1.
+- [operator-sdk](https://github.com/operator-framework/operator-sdk) version v0.15.2.
```

Comment:
The moc is still pointing at 0.15.1
https://github.com/integr8ly/integreatly-operator/blob/a6df320d56ead7e810fd5202416904dd942dec93/go.mod#L25

Reply:
still pointing at v0.15.1 in a few places https://github.com/integr8ly/integreatly-operator/search?q=v0.15.1&unscoped_q=v0.15.1